components/admin_console/save_button.jsx

```diff
@@ -11,26 +11,28 @@ export default class SaveButton extends React.Component {
     static get propTypes() {
         return {
             saving: PropTypes.bool.isRequired,
-            disabled: PropTypes.bool
+            disabled: PropTypes.bool,
+            savingMessageId: PropTypes.string
         };
     }
 
     static get defaultProps() {
         return {
-            disabled: false
+            disabled: false,
+            savingMessageId: 'admin.saving'
```

Comment:
Can you have this default to `setting_item_max.saving` or create a new string for it? I think we'll want to use this elsewhere and having the default text be `Saving` makes more sense than `Saving Config`. You'll need to change it to pass in `admin.saving` in `components/admin_console/admin_settings.jsx` too, but it's just that one place